core/src/main/java/me/aurium/beetle/branch/adapter/ContextAdapter.java

```diff
@@ -0,0 +1,27 @@
+package me.aurium.beetle.branch.adapter;
+
+import me.aurium.beetle.branch.CommandNode;
+import me.aurium.beetle.branch.IdentifiableNode;
+import me.aurium.beetle.branch.block.BlockPath;
+
+/**
+ * cum eternal
+ * @param <T> cum
+ */
+public interface ContextAdapter<T> {
+
+    IdentifiableNode<T> getExecutedNode();
+    CommandNode<T> getBaseExecutedNode();
+
+    BlockPath executedPath();
+
+    //adapter
+
+    T getSender();
+    String getAlias();
+    String[] getArgs();
+    void debugString(String var1);
```

Comment:
What does this do?

Reply:
This adapts the Context from Beetle into a new interface that can do other things. The sender is like the sender of a command, the alias is the alias, and the args are the arguments that the command was called with. This is so that consumers of the context may provide the same functionality as a vanilla Bukkit command

Reply:
Sorry I wasn't clear; I was referring to the debugString method. If you need to debug a string, why not use a logger?

Reply:
Oh, that means: Debug the string **to the sender**, the method name needs to be renamed to reflect that.

Reply:
Resolved